.github/workflows/eslint-prettier-on-pr.yml

```diff
@@ -0,0 +1,31 @@
+# workflow to enforce style and linting on PR
+name: eslint-prettier
+on:
+  pull_request:
+    branches:
+      - main
+      - develop
```

Comment:
Let's drop the develop branch since we haven't actually designated it for anything.